lib/command.js

```diff
@@ -1559,6 +1590,27 @@ Expecting one of '${allowedValues.join("', '")}'`);
     this.error(message, { code: 'commander.missingMandatoryOptionValue' });
   }
 
+  /**
+   * `Option` conflicts with another option.
+   *
+   * @param {Option} option
+   * @param {Option} conflictingOption
+   * @api private
+   */
+  _conflictingOptionPresent(option, conflictingOption) {
```

Comment:
Suggest drop the `Present`

Reply:
Done in https://github.com/tj/commander.js/pull/1678/commits/f7739debcc830d3017c98d4cbfb797814d1e815a